kubeval/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apk add --no-cache git && \
 
 COPY ./analyse /usr/local/bin
 
-ENTRYPOINT ["/usr/local/bin/analyse"]
+CMD ["/usr/local/bin/analyse", "k8s"]
```

Comment:
Hmm... not sure about always assuming the `k8s` folder. I'd keep only the binary, and then in CI it would be used like:

```yaml
kubeval:
  stage: lint-and-build
  image: kiwicom/kubeval
  script:
    - analyse k8s
```

Reply:
Doesn't it make more sense not to specify CMD at all, that running analyse without arguments? Or maybe running `analyse .` as a default command?

Reply:
But `.` is already [the default](https://github.com/kiwicom/dockerfiles/blob/master/kubeval/analyse#L9) `ANALYSEDIR` 🙂

Reply:
You are right.